web/components/command-palette/shortcuts-modal/commands-list.tsx

```diff
@@ -1,39 +1,42 @@
 import { Command } from "lucide-react";
 // helpers
 import { substringMatch } from "@/helpers/string.helper";
+// hooks
+import { usePlatformOS } from "@/hooks/use-platform-os";
 
 type Props = {
   searchQuery: string;
 };
 
-const KEYBOARD_SHORTCUTS = [
-  {
-    key: "navigation",
-    title: "Navigation",
-    shortcuts: [{ keys: "Ctrl,K", description: "Open command menu" }],
-  },
-  {
-    key: "common",
-    title: "Common",
-    shortcuts: [
-      { keys: "P", description: "Create project" },
-      { keys: "C", description: "Create issue" },
-      { keys: "Q", description: "Create cycle" },
-      { keys: "M", description: "Create module" },
-      { keys: "V", description: "Create view" },
-      { keys: "D", description: "Create page" },
-      { keys: "Delete", description: "Bulk delete issues" },
-      { keys: "H", description: "Open shortcuts guide" },
-      {
-        keys: "Ctrl,Alt,C",
-        description: "Copy issue URL from the issue details page",
-      },
-    ],
-  },
-];
-
 export const ShortcutCommandsList: React.FC<Props> = (props) => {
   const { searchQuery } = props;
+  const { platform } = usePlatformOS();
+
+  const KEYBOARD_SHORTCUTS = [
+    {
+      key: "navigation",
+      title: "Navigation",
+      shortcuts: [{ keys: "Ctrl,K", description: "Open command menu" }],
+    },
+    {
+      key: "common",
+      title: "Common",
+      shortcuts: [
+        { keys: "P", description: "Create project" },
+        { keys: "C", description: "Create issue" },
+        { keys: "Q", description: "Create cycle" },
+        { keys: "M", description: "Create module" },
+        { keys: "V", description: "Create view" },
+        { keys: "D", description: "Create page" },
+        { keys: "Delete", description: "Bulk delete issues" },
+        { keys: "H", description: "Open shortcuts guide" },
+        {
+          keys: platform === "MacOS" ? "Ctrl,control,C" : "Ctrl,Alt,C",
```

Comment:
I think there is a typo, it should be CMD+ctrl+c ?
Use consistent language for control